mutis/correlation.py

```diff
@@ -119,14 +138,24 @@ def gen_corr(self):
                     self.times,
                     self.dts,
                 )
-                self.values = kroedel_ab(
-                    self.signal1.times,
-                    self.signal1.values,
-                    self.signal2.times,
-                    self.signal2.values,
-                    self.times,
-                    self.dts,
-                )
+            if uncert is True:
```

Comment:
same as above

Reply:
moreover, this code is duplicated.. consider using a `@staticmethod` function

Reply:
ok, it is not duplicated code... but nearly.. we will refactor the if/elif with a function in a later PR